modules/videoio/src/backend_plugin.cpp

```diff
@@ -645,8 +645,8 @@ class PluginWriter : public cv::IVideoWriter
         if (CV_ERROR_OK != plugin_api_->v0.Writer_write(writer_, img.data, (int)img.step[0], img.cols, img.rows, img.channels()))
         {
             CV_LOG_DEBUG(NULL, "Video I/O: Can't write frame by plugin '" << plugin_api_->api_header.api_description << "'");
+            throw;
```

Comment:
1. `throw` of that?
2. Behavior should be consistent between plugin and builtin mode.
3. Some error are asynchronous. They are not covered here. How to deal with them?

Reply:
Sorry. That's total nonsense. Let me fix that.